packages/amplify-category-function/src/__tests__/provider-utils/awscloudformation/service-walkthroughs/addLayerToFunctionWalkthrough.test.ts

```diff
@@ -6,7 +6,8 @@ import {
   askLayerOrderQuestion,
   askLayerSelection,
 } from '../../../../provider-utils/awscloudformation/utils/addLayerToFunctionUtils';
-import { LayerVersionMetadata, loadLayerDataFromCloud } from '../../../../provider-utils/awscloudformation/utils/layerHelpers';
+import { loadLayerDataFromCloud } from '../../../../provider-utils/awscloudformation/utils/layerHelpers';
```

Comment:
these imports are not right. optimize the path of these, it is probably `../../utils/.....`.

Reply:
This seems fine , I have check the test files and they are working , plus other have same format. since this is a test file ```../```  chain should come to ```provider-utils```

Reply:
The `__tests__` folder in the function category is somewhat deeply nested, this looks fine to me as well. cc @attilah